src/Features/CSharp/Portable/TaskList/CSharpTaskListService.cs

```diff
@@ -46,19 +40,19 @@ internal class CSharpTodoCommentService : AbstractTodoCommentService
                 var index = message.IndexOf(SingleLineCommentPrefix, StringComparison.Ordinal);
                 var start = trivia.FullSpan.Start + index;
 
-                AppendTodoCommentInfoFromSingleLine(commentDescriptors, message.Substring(index), start, todoList);
+                AppendTaskListItemsOnSingleLine(commentDescriptors, document, message.Substring(index), start, items);
```

Comment:
passing `document` along because we use it to encode more information in the final data type we give back.  because this goes through the diagnostics system now, we need to encode into the data type things like the 'mapped file span' and whatnot.